### DISCUSSION TITLE
Items are always null, when building nested navigation

### DISCUSSION THREAD

**Discussion by mcnaveen** [VirtusLab/strapi-plugin-navigation#288] [Q&A]
Hi, I'm trying to set up the Menu. But facing some issues.

Inside the strapi, it looks nested but in the API endpoint, it shows as a separate item. Also, it holds data with a key "parent"
Is it supposed to be inside the items of the wrapper right? Or am I doing something wrong?

In the endpoint, it's not nested at all. 

Here is the API response I'm getting. https://gist.github.com/mcnaveen/6de3d4ffd3cb78f285353b3a4ee6d4cb

Even though I tried to pass the type params as TREE, FLAT all look the same.

![image](https://user-images.githubusercontent.com/8493007/199209726-44d68994-bae7-46e2-9b49-b205cfa92498.png)

**Comment by cyp3rius** [comment]
I cannot reproduce this issue with wrappers. Can you please double check if you're calling it properly? I see response similar to `FLAT` as part of your Gist.

`http://localhost:1337/api/navigation/render/main-navigation?type=TREE`

`http://localhost:1337/api/navigation/render/main-navigation?type=FLAT`
